### PR TITLE
docs/nia: add section on how to update cts-ent license

### PR DESCRIPTION
**Pull request by lornasong**
[link to new section](https://consul-ofofe5suf-hashicorp.vercel.app/docs/nia/enterprise/license#updating-the-license)

Screen Shot 2021-08-27 at 2 50 59 PM

### CODE REVIEW

**Review by findkim — APPROVED** [review]

**Comment by lornasong** [comment]
@trujillo-adam we haven't discussed the best way to get your feedback during the drafting phase yet. Until then, I wanted to flag these latest changes to add documentation on how to update a license. In case it's helpful, I wanted to see if you would like to review before this gets merged into the PR we're currently chatting in. No obligation! Thank you.

File: website/content/docs/nia/enterprise/license.mdx [hunk 1]
```diff
@@ -27,6 +27,16 @@ The options to set the license in order of precedence:
 
 Note: the [options to set the license and order of precedence](/docs/enterprise/license/overview#binaries-without-built-in-licenses) is the same as Consul Enterprise server agents.
 
+### Updating the License
+
+After running Consul-Terraform-Sync Enterprise, your license will eventually approach expiration, and you'll want to update to a new license. To do so, have the new license value at hand or saved in a file to disk.
+
+Steps to update to a new license:
+1. Update the license environment variable or configuration that you used to set the prior license with the new license value or file path
+1. Stop and restart Consul-Terraform-Sync Enterprise
```
Comment (trujillo-adam) [inline website/content/docs/nia/enterprise/license.mdx:36]:
```suggestion
You can set the Consul-Terraform-Sync Enterprise license by either pointing to a license file in your Consul configuration or by specifying the license key in an environment variable. Use the following procedure to update the license when it expires or is near the expiration date: 
1. Update the license environment variable with the new license value or point to the updated license file in your Consul configuration.
1. Stop and restart Consul-Terraform-Sync Enterprise.
```
Please ignore this suggestion if this isn't quite right or doesn't seem like an improvement. I was just trying to see if I could smooth out the wording a bit.

**Review by trujillo-adam — APPROVED** [review]
Please review my suggestion and apply if you think it is an improvement.

File: website/content/docs/nia/enterprise/license.mdx (hunk 1 above)
Comment (lornasong) [reply website/content/docs/nia/enterprise/license.mdx]: I think this is an improvement to the language. I ended up doing a combination to use your improvements and clarify some details about the configuration and relation to other content (not in this PR). I appreciate the suggestion. Thank you

**Pull request merged**